thoth/metrics_exporter/jobs.py

```diff
@@ -200,3 +141,77 @@ def get_python_packages_solver_error_count():
     except aiohttp.client_exceptions.ClientConnectorError as excptn:
         graphdb_connection_error_status.set(1)
         _LOGGER.error(excptn)
+
+
+def get_unique_python_packages_count():
+    """Get the total number of unique python packages in Thoth Knowledge Graph."""
+    try:
+        graph_db = GraphDatabase()
+        graph_db.connect()
+
+        total_unique_python_packages = len(graph_db.get_python_packages())
+
+        graphdb_total_unique_python_packages.set(total_unique_python_packages)
+
+        _LOGGER.debug("graphdb_total_unique_python_packages=%r", len(graph_db.get_python_packages()))
+
+    except aiohttp.client_exceptions.ClientConnectorError as excptn:
```

Comment:
Do we encounter these exception types or is that a left-over from JanusGraph? I would say we now need to handle gRPC related exceptions as we use gRPC.